Header header

# Новое значение магнитного склонения
float32 declination
